interactive_engine/src/data_load_tools/README.md

```diff
@@ -0,0 +1,93 @@
+# Data Load Tools
+This is a toolset for bulk loading data from raw files to graphscope persistent storage service. It includes a hadoop map-reduce job for offline data build, and a tool for ingesting and committing the offline output to the persistent storage service.
+
+## Quick Start
+### Prequisities
+
+- Java compilation environment (Maven 3.5+ / JDK1.8), if you need to build the tools from source code
+- Hadoop cluster (version 2.x) that can run map-reduce jobs and has HDFS supported
+- Running GIE with persistent storage service (graph schema should be properly defined)
+
+### Get Binary
+
+If you have the distribution package `maxgraph.tar.gz`, decompress it. Then you can find the map-reduce job jar `data_load_tools-0.0.1-SNAPSHOT.jar` under `maxgraph/lib/` and the executable `load_tool.sh` under `maxgraph/bin/`. The `load_tool.sh` is just a wrapper for java command, you can only use `data_load_tools-0.0.1-SNAPSHOT.jar` in the following demonstration. 
+
+If you want to build from source code, just run `mvn clean package -DskipTests`. You can find the compiled jar `data_load_tools-0.0.1-SNAPSHOT.jar` in the `target/` directory.
```

Comment:
if user choose to build the source code, where to find the file `load_tool.sh`?
maybe also place the file in this folder?

Reply:
load_tool.sh is not neccessary for bulk loading. It's just a java wrapper for convienience. I have only put it in the distribution package because we have to deal with different jar path otherwise, which seems unneccessary. User can use only data_load_tools-0.0.1-SNAPSHOT.jar to accomplish the data loading process.

Reply:
I see. It is optional.